tests/subsys/fs/fat_fs_api/src/test_fat.h

```diff
@@ -9,7 +9,17 @@
 #include <zephyr/ztest.h>
 #include <zephyr/fs/fs.h>
 
-#define FATFS_MNTP	"/NAND:"
+#ifdef CONFIG_DISK_DRIVER_RAM
+#define DISK_NAME CONFIG_DISK_RAM_VOLUME_NAME
+#elif defined(CONFIG_DISK_DRIVER_FLASH)
+#define DISK_NAME DT_PROP(DT_NODELABEL(test_disk), disk_name)
+#elif defined(CONFIG_DISK_DRIVER_SDMMC)
+#define DISK_NAME SDMMC_VOLUME_NAME
+#else
+#error "Failed to select DISK access type"
+#endif
+
+#define FATFS_MNTP	"/"DISK_NAME":"
```

Comment:
For lfs-based tests and samples (including smp_svr) mount points are just using /<name/ without a colon e.g. /lfs1/ or /lfs3/ - this is the same style as device numbering for e.g. i2c and spi: i2c0, i2c1, spi3, spi82, etc. So with that, do we want to have FAT be the odd one out?

Reply:
The FAT has not the best way of figuring out paths and the ":" is required unless me modify FAT zephyrproject-rtos/fatfs, which is slightly modified (just to make it work with us) ELM FAT driver.